src/pages/home/report/ContextMenu/BaseReportActionContextMenu.js

```diff
@@ -115,6 +154,7 @@ function BaseReportActionContextMenu(props) {
                             onPress={() => interceptAnonymousUser(() => contextAction.onPress(closePopup, payload), contextAction.isAnonymousAction)}
                             description={contextAction.getDescription(props.selection, props.isSmallScreenWidth)}
                             isAnonymousAction={contextAction.isAnonymousAction}
+                            focused={focusedIndex === index}
```

Comment:
I also thought `isFocused` name is better but didn't request change because `focused` prop name already exists here 😄